components/prombench/manifests/benchmark/3_prometheus-test.yaml

```diff
@@ -736,7 +736,10 @@ spec:
       initContainers:
       - name: prometheus-builder
         image: docker.io/prombench/prometheus-builder:2.0.1
-        args: ["{{ .PR_NUMBER }}"]
+        args: [
+          "{{ .PR_NUMBER }}",
+          "/prometheus-builder", # should be same as mountPath
```

Comment:
I would prefer named variables so that changing the order won't make it break.

PR_NUMBER="{{ .PR_NUMBER }}"
VOLUME_DIR="/prometheus-builder", # should be same as mountPath